### PR TITLE
Add sumologic-livetail-cli

### PR DESCRIPTION
**Pull request by rjcoelho**
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/sumologic-livetail-cli.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'sumologic-livetail-cli' do
+  version '1.0'
+  sha256 '40cc322b1f31ab9189798d4bb193d1eb43d00c95906bdb863dadefc72f4a839c'
+
+  url 'https://github.com/SumoLogic/livetail-cli/releases/download/v1.0/livetail_mac_os.zip'
```
Comment (vitorgalvao) [inline Casks/sumologic-livetail-cli.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

File: Casks/sumologic-livetail-cli.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+cask 'sumologic-livetail-cli' do
```
Comment (vitorgalvao) [inline Casks/sumologic-livetail-cli.rb]: Just `livetail`.

Comment (rjcoelho) [inline Casks/sumologic-livetail-cli.rb]: You mean livetail or sumologic-livetail ?

File: Casks/livetail.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+cask 'livetail' do
+  version '1.0'
+  sha256 '40cc322b1f31ab9189798d4bb193d1eb43d00c95906bdb863dadefc72f4a839c'
+
+  url 'https://github.com/SumoLogic/livetail-cli/releases/download/v1.0/livetail_mac_os.zip'
+  appcast 'https://github.com/SumoLogic/livetail-cli/releases.atom', checkpoint: 'beadbf08fd78c786ffad244d9cd442013797505862f5d11975a48257476c486c'
```
Comment (vitorgalvao) [inline Casks/livetail.rb]: Did you run `brew cask style --fix` after this change? Because this needs to be two lines.

File: Casks/livetail.rb [hunk 4]
```diff
@@ -0,0 +1,12 @@
+cask 'livetail' do
+  version '1.0'
+  sha256 '40cc322b1f31ab9189798d4bb193d1eb43d00c95906bdb863dadefc72f4a839c'
+
+  url 'https://github.com/SumoLogic/livetail-cli/releases/download/v1.0/livetail_mac_os.zip'
```
Comment (vitorgalvao) [inline Casks/livetail.rb]: `url "https://github.com/SumoLogic/livetail-cli/releases/download/v#{version}/livetail_mac_os.zip"`

**Pull request merged**